### PR TITLE
Bounce handling via SNS

### PR DESCRIPTION
**Pull request by jbarrett**
##### Description :

Service to remove subscribers, acting upon bounce notifications from SNS.

##### Reviewer notes :

- Schema change in sql/1.203/bounces.sql
- ...

##### Who should be informed of this change?

@yegg @AdamSC1-ddg 

##### Does this change have significant privacy, security, performance or deployment implications?

This PR introduces additional facilities to verify users and respect user privacy.

New CPAN dependencies:

```
+requires 'Ouch', '0.0410';
+requires 'Crypt::OpenSSL::X509' '1.804';
+requires 'Crypt::OpenSSL::Random' '0.11';
+requires 'Crypt::OpenSSL::RSA' '0.28';
+requires 'AWS::SNS::Verify' '0.0102';
```

New service definition will be created elsewhere.

##### Checklist :
- [x] Back end tests (perl, scripts)
- [ ] Front end tests (js, integration)
- Browser verification
    - [ ] IE
    - [ ] Chrome
    - [ ] Firefox
    - [ ] Safari
    - [ ] Opera 
- Mobile verification
    - [ ] iOS
    - [ ] Android

### CODE REVIEW

File: cpanfile [hunk 1]
```diff
@@ -133,4 +133,8 @@ requires 'HTML::FormatText::WithLinks', '0.15';
 requires 'Cache::LRU', '0.04';
 requires 'Number::Format', '1.73';
 requires 'Starman', '0.4009';
-
+requires 'Ouch', '0.0410';
+requires 'Crypt::OpenSSL::X509' '1.804';
+requires 'Crypt::OpenSSL::Random' '0.11';
+requires 'Crypt::OpenSSL::RSA' '0.28';
+requires 'AWS::SNS::Verify' '0.0102';
```
Comment (jbarrett) [inline cpanfile]: New dependencies

File: cpanfile [hunk 2]
```diff
@@ -133,4 +133,8 @@ requires 'HTML::FormatText::WithLinks', '0.15';
 requires 'Cache::LRU', '0.04';
 requires 'Number::Format', '1.73';
 requires 'Starman', '0.4009';
-
+requires 'Ouch', '0.0410';
```
Comment (MariagraziaAlastra) [inline cpanfile]: "Ouch"? 😃 
Sorry couldn't resist

**Comment by MariagraziaAlastra** [comment]
LGTM 👍

**Comment by MariagraziaAlastra** [comment]
Thanks!

**Pull request merged**